src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -421,6 +481,49 @@ const SearchFieldList = ({ onSearch }: Props) => {
                     select
                     disabled={!searchParams.includeStudent}
                   />
+                  {switchYearRange == true ? (
+                    <SearchField
+                      name="graduation_year"
+                      value={searchParams.graduation_year}
+                      updateValue={handleUpdate}
+                      options={Array.from({ length: userProvidedYear - 1889 + 1 }, (_, i) => ({
+                        value: (userProvidedYear - i).toString(),
+                        label: (userProvidedYear - i).toString(),
+                      }))}
+                      Icon={FaCalendarTimes}
+                      disabled={!searchParams.includeAlumni}
+                      select
+                    />
+                  ) : (
+                    <Grid item width={225}>
+                      <Slider
```

Comment:
I like the idea of a slider here, but I'm concerned it may be the wrong choice in terms of UX. Since the range is several hundred years, it takes a lot of effort to get the value exactly what you want it to be. Consider someone on a touch screen who has slightly shaky hands - they would probably have great difficult setting this to be exactly 1970-1974, for example. I think in the interest of UX, we might need to change this to two separate year inputs.

Reply:
I agree with you the slider is probably not the best option for this feature. We were thinking of some options and wanted your opinion. 

1. Keeping the slide, but also allowing some other way of changing the year range.
2. Making a select that allow you to choose two dates for the range
3. Making two text boxes for the initial and end year (Not my preferred option since people might type other things and it might not be intuitive for the user)

Reply:
I honestly don't know if there is a really good option. I think in terms of mobile usability I would argue that a location where we could select a year would be optimal. However, I don't know if the practicality of a year selector is much better (if any better) than the live updating of the slider.